terminal.py

```diff
@@ -75,11 +75,11 @@ def __init__(self, jobs_cmds: List[str] = None):
 
         if jobs_cmds:
             # close the eyes if the user forgets the initial `/`
-            if len(jobs_cmds) > 0:
-                if jobs_cmds[0][0] != "/":
-                    jobs_cmds[0] = f"/{jobs_cmds[0]}"
+            for cmd_i, job_cmd in enumerate(jobs_cmds):
+                if not job_cmd.startswith("/"):
+                    jobs_cmds[cmd_i] = f"/{job_cmd}"
```

Comment:
I don't think we even need to have `cmd_i`, couldn't we just use `jobs_cmds.append(f"/{job_cmd}")`?

Reply:
mhmm, I think in this case we need it. 
What about this command (just an example cmds don't necessarily make sense):

``` python terminal stocks/disc/gainers economy /stocks/exit```

we have 3 cmds:

1.`stocks/disc/gainers`
2.`economy`
3.`/stocks/exit`

On the first 2 we wanna add the / and on the last one we don't.

Lemme know what you think

Reply:
I see! I missed that the same list was being used. All good for me!

You could also have done it through a one-liner:
```
jobs_cmds = [job_cmd if job_cmd.startswith("/") else f"/{job_cmd}" for job_cmd in jobs_cmds]
```
😄

Reply:
Ah yes, even better!